RevenueCatUI/Templates/V2/Components/Text/TextComponentView.swift

```diff
@@ -114,7 +114,7 @@ struct TextComponentView_Previews: PreviewProvider {
                     component: .init(
                         text: "id_1",
                         color: .init(light: .hex("#000000")),
-                        fontSize: .headingXXL
+                        fontSize: 40
```

Comment:
Nit: It'd be nice if all of these could be hardcoded somewhere, like using the `PaywallComponent.FontSize.size` property defined further down in the PR, but since these are just previews, it's probably okay!